Source/BestProjectEver/Public/Character/BPE_Enemy.h

```diff
@@ -83,5 +97,8 @@ class BESTPROJECTEVER_API ABPE_Enemy : public ABPE_BaseCharacter
 	virtual FRotator GetViewRotation() const override;
 
 	/** set by the AI controller if an actor is perceived */
-	void SetTargetViewLocation(FVector TargetLocation) { TargetViewLocation = TargetLocation; };
+	void SetTargetViewLocation(FVector TargetLocation) { TargetViewLocation = TargetLocation; }
```

Comment:
maybe let the FVector param be a const reference?

Reply:
![image](https://user-images.githubusercontent.com/85327912/173151629-8562576c-614e-40f3-8e81-7be0398124d8.png)

Reply:
![image](https://user-images.githubusercontent.com/85327912/173152644-ce7fb11b-c06f-496b-97c8-9c7b38e4f110.png)